src/scripts/browser.js

```diff
@@ -116,7 +116,9 @@ function web0sVersion(browser) {
 
         // The next is only valid for the app
 
-        if (browser.versionMajor >= 79) {
+        if (browser.versionMajor >= 94) {
+            return 8;
```

Comment:
Isn't it called `23`?

https://webostv.developer.lge.com/develop/specifications/web-api-and-web-engine

Also, please add webOS 22 (`browser.versionMajor >= 87`).

Reply:
Done in 47c86301a1be056307afb53b3739447190adee2d